src/Makefile.common

```diff
@@ -70,6 +70,7 @@ realclean::	clean
 $(OBJS): $(top_builddir)/src/include/version.hh \
 		$(top_builddir)/src/plugin/include/plugin_config.hh \
 		$(top_builddir)/src/base/bios/bios_offsets.hh \
+		$(top_builddir)/src/base/bios/bios_data.xxd \
```

Comment:
Same, this is unneeded, as they do not
depend on it directly. You should only write
direct deps.